src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-Avatar/Components/Avatar.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import emotionSerializer, { matchers } from 'jest-emotion';
+import { Avatar } from './Avatar';
+import { AvatarImage } from '../styles';
+import { avatar } from '../types';
+
+const pre = 'https://storage.googleapis.com/ignitus_assets/ig-avatars/';
+
+const avatarNames = ['grant'];
```

Comment:
Use array given in ```avatars.ts```  file.

Reply:
Done

Reply:
My motive was to use directly from it instead of rewriting, well no issue.